### ISSUE TITLE
IFCHAR = T ??

### ISSUE THREAD

**Issue by fischer1** [Nek5000/NekExamples#100]
I notice that several examples have ifchar=T.

Some don't even use a dt big enough to warrant such usage (e.g., cyl_restart case).

I think all examples (except for short tests designed to test Characteristics) should have IFCHAR F unless we really want to show off CFL >> 1.

Otherwise, jobs will run twice as slow.   People often start with one of the examples and modify it to suit their needs without realizing that IFCHAR=T.

(This happened in one of the large KTH runs that George did for pipe flow.  He was running with CFL=0.5 and IFCHAR T.   Turning that to F sped the code up by 2X and would have saved a million core hours---it was my fault for not paying attention earlier that time.)

Can we / should we switch all the examples to have default IFCHAR F   (since the decision
to use Characteristics is typically a more advanced-user decision) ?

**Comment by stgeke** [comment]
I agree but a few examples (like turbChannel) should demonstrate how to use CHAR.

-----Original message-----

**Comment by fischer1** [comment]
Yes - we definitely want examples that demo ifchar.

I was just surprised (and I'm sure it's my fault) that ext_cyl had IFCHAR T yet the CFL was 0.5

So, I grepped "IFCHAR" in all the */*.rea files and saw that there were other places
where it was true.    Given that the overhead his potentially expensive, we probably
want to notify users which cases have IFCHAR T --- turbchannel is a good one.

**Comment by adampep** [comment]
Right now there are 5 examples using characteristics on purpose: Pipe_Stenosis, TurbJet, TurbChannel, TurbPipe and Vortex. I removed the rest of the flags except Os7000, as in this case delta in the test is so small, that the test crushes even though cfl=0.475. What was the source of the test value in this case?
